README.md

```diff
@@ -82,6 +82,20 @@ WARNING! this will erase the catalog, and thus require re-seeding from scratch.
   Basically an especially inconvenient confirmation dialogue.  For safety's sake, the full command that skips that warning can be constructed by the user as needed, so as to prevent unintentional copy/paste dismissal when the user might be administering multiple deployment environments simultaneously.  Inadvertent database wipes are no fun.
   * `db:reset` will make sure db is migrated and seeded.  If you want to be extra sure: `RAILS_ENV=[environment] bundle exec rake db:migrate db:seed`
 
+#### Drop or populate the catalog for a single endpoint
+
+There are rake tasks for clearing out all database entries for a single endpoint. To run either of the rake tasks, invoke them as follows, prepending the `RAILS_ENV` and giving the name of the endpoint as an argument (note that the argument is not a string):
```

Comment:
... (note that the argument is not in quotes because it is not a string) ...   ???

Reply:
yeah maybe it's not needed and is a given.

Reply:
i can file a ticket for this - since it is already merged to master